src/common/rabbitmq/entities/nft/nft.topics.ts

```diff
@@ -0,0 +1,24 @@
+import { BinaryUtils } from "src/utils/binary.utils";
+import { NumberUtils } from "src/utils/number.utils";
+
+export class NftTopics {
+  identifier: string = '';
+  collection: string = '';
+  nonce: number = 0;
+
+  constructor(rawTopics: string[]) {
+    this.collection = BinaryUtils.base64Decode(rawTopics[0]);
```

Comment:
Can you surround the decoding part with try catch and check if rawTopics[0] & rawTopics[1] actually exists?
So that if an error occurs you can log something like this 'Unexpected error when decoding NFT topics from raw topics'